.travis.yml

```diff
@@ -1,28 +1,40 @@
 language: node_js
 node_js:
-- '7'
+  - '7'
```

Comment:
I like this because it's non-ambiguous and minimal.  Shouldn't need nvm when you're automating your infrastructure, IMO.